ui/src/app/kvm/components/KVMResourcesCard/KVMResourcesCard.tsx

```diff
@@ -56,10 +57,40 @@ const KVMResourcesCard = ({
       <div className="kvm-resources-card__section kvm-resources-card__ram">
         <div>
           <h4 className="p-heading--small">RAM</h4>
-          <div className="kvm-resources-card__ram-chart">
-            {ram.general.total}
-            {ram.general.unit}
-          </div>
+          <DoughnutChart
+            label={`${ram.general.total + (ram.hugepage?.total || 0)}${
+              ram.general.unit
+            }`}
+            segmentHoverWidth={20}
+            segmentWidth={15}
+            segments={[
+              {
+                color: "#06C",
+                tooltip: `General allocated ${ram.general.allocated}`,
```

Comment:
The tooltip values are missing units.

Now that I'm looking at it, it probably makes sense to just have one unit for RAM instead of one for general and one for hugepage, but that's neither here nor there because it'll be replaced by real data anyway. :man_shrugging: